Flash Chat iOS13/Controllers/ThreadListViewController.swift

```diff
@@ -108,13 +107,30 @@ class ThreadListViewController: UITableViewController {
 	override func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
 		if editingStyle == .delete {
 			// Delete the row from the data source
-			database.collection(Constants.FStore.threadsCollectionName).getDocuments() { (querySnapshot, error) in
+			database.collection(Constants.FStore.threadsCollectionName).getDocuments() { (threadQuerySnapshot, error) in
 				if let error = error {
 					AppDelegate.showError(error, inViewController: self)
 				} else {
-					if let threads = querySnapshot?.documents {
+					if let threads = threadQuerySnapshot?.documents {
 						for thread in threads {
 							if thread == threads[indexPath.row] {
+								self.database.collection(Constants.FStore.threadsCollectionName).document(thread.documentID).collection(Constants.FStore.bubblesField).getDocuments { bubbleQuerySnapshot, error in
+									if let error = error {
+										fatalError("Can't get bubbles: \(error)")
```

Comment:
Still recommend that this and line 131 are not fatal errors, but will not block merge because of it